code/__DEFINES/_globals.dm

```diff
@@ -20,11 +21,16 @@
 #define GLOBAL_PROTECT(X)
 #endif
 
-/// Standard BYOND global, do not use
-#define GLOBAL_REAL_VAR(X) var/global/##X
+/// Standard BYOND global, seriously do not use without an earthshakingly good reason
+#define GLOBAL_REAL_VAR(X) var/global/##X;
 
-/// Standard typed BYOND global, do not use
-#define GLOBAL_REAL(X, Typepath) var/global##Typepath/##X
+// ▲▼ I added the semi-colons because people kept using these to fuck with init order. See /code/game/world.dm for that - Dominion/Cyberboss
+
+/// Standard typed BYOND global, seriously do not use without an earthshakingly good reason
+#define GLOBAL_REAL(X, Typepath) var/global##Typepath/##X;
+
+/// !!!!!!USED ONLY BY MASTER ONLY BY MASTER ONLY BY MASTER DO NOT FUCKING USE THIS!!!!!!
+#define MASTER_GLOBAL var/global/datum/controller/master/Master = world.RealGlobalsPreInit();
```

Comment:
Why are we putting this here to use in one file instead of putting it in the master file? The master controller is already treated as a pretty unique piece of machinery

Reply:
Consistency with other `/var/global`s? Idk. I'll move it

Reply:
Wait, I remember why, this is explicitly implementing a macro that overrides the linter that checks for real global usage
(top-level `var/`s). I wanted to put it in this file because nothing in it should be emulated elsewhere.

Reply:
Sounds ok, should be added to the comment though because that's theoretically reasonably fixable

Reply:
This was changed